app/components/ActionLinks/index.js

```diff
@@ -43,23 +43,6 @@ const ActionLinks = ({
   pecsUrl,
   isPomAllocUser,
 }) => {
-  if (
-    !isKeyWorker &&
-    !isKeyWorkerAdmin &&
-    !isWhereabouts &&
-    !isEstablishmentRollCheck &&
-    !hasAdminRights &&
-    !isGlobalSearch &&
-    !isAddBulkAppointments &&
-    !isCatToolUser &&
-    !isUseOfForce &&
-    !isPathfinderUser &&
-    !isPomAllocUser &&
-    !isLicenceUser &&
-    !isPecsUser
-  ) {
-    return <div />
-  }
   return (
```

Comment:
46-62 removed as Covid button will always be visible

Reply:
in hindsight, the way this was modelled originally could have been better:

rather than: 

```
if (!a && !b && !c && !d && !e && !f) {
  return <div/ >
}

if (a) {
  <comp 1>
}
if (b) {
  <comp 2>
}
if (c) {
  <comp 3>
}
...
```

it would have nicer to have done: 
```
const actions = getActions() // returns [ [a && ...[<comp 1>]], [b &&  ...[<comp 2>]], ... ]]
if (!actions.length) {
   <div />
} else {
   render actions
}
```